carcara/src/parser/error.rs

```diff
@@ -81,8 +81,12 @@ pub enum ParserError {
     #[error("expected {0} arguments, got {1}")]
     WrongNumberOfArgs(Range, usize),
 
+    /// The argument values are not in the expected range.
     #[error("expected argument value to be greater than {0}, got {1}")]
-    ExpectedGreaterThanArgs(usize, usize),
+    ExpectedGreaterThanArgs(Range, usize),
```

Comment:
Since this now accepts any `Range`, I think the name is a bit confusing. It would be clearer if it were something like
```suggestion
    #[error("expected argument value to be {0}, got {1}")]
    WrongValueOfArgs(Range, usize),
```